prime-router/src/test/resources/fhirengine/smoketest/Expected_HL7_to_FHIR_FULLELR.fhir

```diff
@@ -8024,6 +8024,42 @@
           }
         ],
         "status" : "final",
```

Comment:
Just adding in data that was dropped prior to this PR. Our FHIR data compare class didn't really care about this difference either.